Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -76,9 +77,26 @@ public static void Init(Context activity, Bundle bundle, Assembly resourceAssemb
 			SetupInit(activity, resourceAssembly);
 		}
 
+		/// <summary>
+		/// Sets title bar visibility programmatically. Must be called after Xamarin.Forms.Forms.Init() method
+		/// </summary>
+		/// <param name="visibility">Title bar visibility enum</param>
 		public static void SetTitleBarVisibility(AndroidTitleBarVisibility visibility)
 		{
+			if((Activity)Context == null) throw new NullReferenceException("Must be called after Xamarin.Forms.Forms.Init() method");
+
 			TitleBarVisibility = visibility;
+
+			if (TitleBarVisibility == AndroidTitleBarVisibility.Never)
+			{
+				if(!((Activity)Context).Window.Attributes.Flags.HasFlag(WindowManagerFlags.Fullscreen))
+					((Activity)Context).Window.AddFlags(WindowManagerFlags.Fullscreen);
```

Comment:
I'm actually wondering if we should take the new `WindowsSoftInputModeAdjust` Platform Specific option into consideration while we're setting this option and decide if there is a conflict, and if so, which feature should win. See 6f507002d26f14004fe54bc1e989b15c5b71c3da.